src/test/java/com/puppycrawl/tools/checkstyle/internal/XpathRegressionTest.java

```diff
@@ -102,13 +102,20 @@
                     "AtclauseOrder",
                     "JavadocBlockTagLocation",
                     "JavadocParagraph",
+                    "JavadocStyle",
                     "JavadocTagContinuationIndentation",
                     "MissingDeprecated",
                     "NonEmptyAtclauseDescription",
                     "SingleLineJavadoc",
                     "SummaryJavadoc"
     )));
 
+    // Older regex-based checks that are under INCOMPATIBLE_JAVADOC_CHECK_NAMES
+    // but not subclasses of AbstractJavadocCheck.
+    // Will eventually include JavadocMethodCheck and JavadocTypeCheck.
+    private static final Set<Class<?>> ADDITIONAL_JAVADOC_CHECKS =
```

Comment:
`ADDITIONAL` does not really explain what this is. How about `REGEXP_JAVADOC_CHECKS`?

Reply:
Sounds good, changed